test/data/models/product.rb

```diff
@@ -0,0 +1,3 @@
+class Product < ActiveRecord::Base
```

Comment:
none of the other models had timestamps on. The timestamps are needed to exhibit the fact that the `cache_key` changes on modification of the translation. 

I considered adding timestamps to an existing model but I was concerned about unintended side-effects, so I opted for added a new model instead.

I'll add note about this to the commit message.